lang/lang-es.js

```diff
@@ -0,0 +1,593 @@
+    'light-blue-3': 'light blue 3',
+    'light-blue-2': 'light blue 2',
+    'light-blue-1': 'light blue 1',
+    'dark-blue-1': 'dark blue 1',
+    'dark-blue-2': 'dark blue 2',
+    'dark-blue-3': 'dark blue 3',
+    'light-purple-3': 'light purple 3',
+    'light-purple-2': 'light purple 2',
+    'light-purple-1': 'light purple 1',
+    'dark-purple-1': 'dark purple 1',
+    'dark-purple-2': 'dark purple 2',
+    'dark-purple-3': 'dark purple 3',
+    'light-magenta-3': 'light magenta 3',
+    'light-magenta-2': 'light magenta 2',
+    'light-magenta-1': 'light magenta 1',
+    'dark-magenta-1': 'dark magenta 1',
+    'dark-magenta-2': 'dark magenta 2',
+    'dark-magenta-3': 'dark magenta 3',
+    Shapes: 'Formas',
+    disableAllChat: 'Desactivar chat privado',
+    enableAllChat: 'Activar chat privado',
+    disableAllGroupChat: 'Desactivar chat común',
+    enableAllGroupChat: 'Activar chat común',
+    disableAllRaisehand: 'Desactivar manos levantadas',
+    enableAllRaisehand: 'Activar manos levantadas',
+    disableAllAskQuestion: 'Desactivar plantear pregunta',
+    enableAllAskQuestion: 'Activar plantear pregunta',
+    disableAllQaMarkNotes: 'Desactivar marcas y notas',
+    enableAllQaMarkNotes: 'Activar marcas y notas',
+    disableAllUserlist: 'Deshabilitar lista de usuarios',
+    enableAllUserlist: 'Habilitar lista de usuarios',
+    Audio: 'Audio',
+    Video: 'Vídeo',
+    Chat: 'Chat privado',
+    GroupChat: 'Chat común',
+    RaiseHand: 'Levantar mano',
+    UserList: 'Lista de usuarios',
+    uploading: 'Procesando...',
+    userListHeader: 'Usuarios ',
+    commonChatHeader: 'Sala de chat',
+    teacherVideoHeader: 'Vídeo del docente',
+    askQuestionHeader: 'Plantear una duda',
+    questionLable: 'Preguntar',
+    answerLable: 'Responder',
+    commentLable: 'Comentar',
+    markAnswer: 'Marcar como respuesta',
+    markNotes: 'Marcas y notas',
+    askQuestionTimeExceed: 'No se puede editar o eliminar, se ha excedido el límite de tiempo',
+    upvoted: 'No se puede editar o eliminar, se ha agregado un voto positivo',
+    enterText: 'Introduzca el texto antes de publicar',
+    more: '...Más',
+    less: 'Menos',
+    askQuestion: 'Plantear la pregunta',
+    MarksAndNotes: 'Marcas y notas',
+    markAnswerUnmark: 'Primero debe desmarcar la respuesta ya marcada',
+    /* For Chrome */
+    PermissionDeniedErrorExt: '<div class="errorMsg"> Se ha bloqueado el acceso a la cámara.'
+    + ' Para proporcionar acceso a la cámara web, siga los siguiente pasos <br />'
+    + 'Ir al ícono de la cámara en la esquina superior derecha de la pantalla <br /> '
+    + 'Haga clic en "Permitir siempre..." y seleccione la opción de cámara <br />  '
+    + 'Acepte y actualice la pantalla  </div> <div class="screenImages">'
+    + ' <figure class="chrome" > <img src="https://www.congrea.com/wp-content/uploads/2016/10/ff-video.png" > <figcaption>Chrome</figcaption> </figure></div>',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 159. Maximum allowed is 120.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5529949)